api/src/log.rs

```diff
@@ -79,7 +84,7 @@ pub async fn middleware_log<E: Endpoint>(next: E, request: Request) -> Result<Re
     // Push a counter based on the request source, sliced up by endpoint + method.
     REQUEST_SOURCE_CLIENT
         .with_label_values(&[
-            determine_request_source_client(&log.user_agent),
+            determine_request_source_client(&log.aptos_client),
```

Comment:
are we already using the user agent in other cases? do we need to keep both online for some period of time during the transition?

Reply:
I think it could be useful in general for just viewing clients that aren't created by us.  So, we can revisit this later.

let's also add a way to set it in the CLI / Rust SDK / Python SDK